test/preparation.js

```diff
@@ -11,6 +11,11 @@ if (!exports.conversation) {
             const user2 = await client2.logon();
             const user3 = await client3.logon(config.sdktester1.credentials);
             module.exports.conversation = await client.createGroupConversation([user2.userId], `globalTestConversation${Date.now()}`);
+            const existingConversations = await client.getConversations({ numberOfConversations: 10 });
+            // create up to 10 past conversations to be used for testing
+            for (let i = 0; i < 10 - existingConversations.length; i++) {
+                await client.createConferenceBridge(`newConversation${i + 1}`);
```

Comment:
We could create them in parallel instead of waiting after each one. But its fine like this since this is only done the first time when using a new bot.